core-schema/src/schema.rs

```diff
@@ -0,0 +1,585 @@
+//! A GraphQL schema referencing one or more specifications
```

Comment:
It looks like this PR is missing all logic relating to export. Though it would be fine to merge it and add export logic as a follow-up PR.

Reply:
Ah I guess it also doesn't have a facility to find spec-specific elements, prefixing logic, etc. Can certainly be a follow-up PR!